spec/support/active_record/models.rb

```diff
@@ -0,0 +1,24 @@
+class Event < ActiveRecord::Base
+  self.primary_key = "identifier"
```

Comment:
Is it required for a Model that uses `acts_as_hypertable` to also set the primary key to `identifier`? Or are these (and the example in the README) just for demonstration purposes?

Reply:
Great question! And one I'd be terrible at answering so I'll shoot you over to the docs: https://docs.timescale.com/timescaledb/latest/quick-start/ruby/#step-3-create-a-table

The **TL;DR** is that when using Timescale (and when your model is a hypertable), you don't use a "traditional" `id` primary key so you have to set it yourself. In all our examples it just so happens to be "identifier". In my personal project, my primary key is "name"